orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5811,21 +5814,21 @@ def test_elementwise(self, input_ids):
             return output
 
         # test MatMul op for padding elimination
-        # in case 0, the shapes of inputs of MatMul are [2, seqlen] and [batch_size, seqlen, hidden_size]
-        #            this case is not support in padding elimination, so the MatMul should not be included in padding
-        #            elimination subgraph and the GatherGrad should be added before MatMul.
-        # in case 1, the shapes of inputs of MatMul are [batch_size, seqlen, hidden_size] and [hidden_size, 128]
+        # in case 0, the shapes of inputs of MatMul are [batch_size, seqlen, hidden_size] and [hidden_size, 128]
```

Comment:
can you add explaination for case 0/1 for non-matmul ops?

Reply:
Only Elementwise ops and 'MatMul', 'ReduceMean' have multiple cases and have added comments for all them.